Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -267,12 +267,10 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
 // MARK: - Functions
 
 private func initialPledgeAmount(
-  from project: Project,
+  from _: Project,
   currentAmount: Double,
   minValue: Double
 ) -> Double {
-  guard userIsBackingProject(project) else { return minValue }
-
```

Comment:
Why don't we need this guard here anymore?

Reply:
I don't think we ever needed this guard. It looks like it's here to protect against developer error more than anything. (If the user isn't backing the project, we always set bonus support to `0` to start.)

The reason I removed it is because the thing it's saying ("if the user isn't already backing the project, change the bonus support to start at the min value") is wrong in a world where the confirm details screen uses the bonus support set on the add-ons screen as its initial value. Keeping it would reset the bonus support we see on the confirm details page. Deleting this guard means that the confirm details page, while not actually providing value to the UI, isn't doing anything silly either.

I'm okay with reverting this change and keeping the guard if you'd rather! But note that if we do that, then getting rid of bonus support on the confirm details page (or getting rid of the entire confirm details page) is definitely launch blocking.